test/1_stdlib/PrintFloat.swift

```diff
@@ -191,17 +209,48 @@ PrintTests.test("Printable") {
   expectDebugPrinted("1.24999998e+17", asFloat32(125000000000000000.0))
   expectDebugPrinted("1.25", asFloat32(1.25))
   expectDebugPrinted("1.24999997e-05", asFloat32(0.0000125))
+  expectDebugPrinted("inf", Float.infinity)
+  expectDebugPrinted("-inf", -Float.infinity)
+  expectDebugPrinted("nan", Float.nan)
+  expectDebugPrinted("-nan", -Float.nan)
+  expectDebugPrinted("nan(0xffff)", Float(nan: 65535, signaling: false))
+  expectDebugPrinted("-nan(0xffff)", -Float(nan: 65535, signaling: false))
+#if !arch(i386)
+  expectDebugPrinted("snan", Float.signalingNaN)
+  expectDebugPrinted("snan(0xffff)", Float(nan: 65535, signaling: true))
+  expectDebugPrinted("-snan(0xffff)", -Float(nan: 65535, signaling: true))
```

Comment:
Could you add tests with the largest possible payload, to make sure that high bits are not dropped in any of the conversions?